main_train_psnr.py

```diff
@@ -47,7 +80,14 @@ def main(json_path='options/train_msrresnet_psnr.json'):
 
     opt = option.parse(parser.parse_args().opt, is_train=True)
     opt['dist'] = parser.parse_args().dist
-
+    if int(os.environ.get('LOCAL_RANK', 0)) == 0:
```

Comment:
instead of checking if local rank is 0 twice just put it in a properly named variable